ipapython/admintool.py

```diff
@@ -247,6 +248,8 @@ def setup_logging(self, log_file_mode='w'):
                 root_logger.removeHandler(handler)
                 break
 
+        if self.log_file_name:
+            self.log_file_initialized = True
```

Comment:
The attribute `self.log_file_initialized` should be set to True only after `standard_logging_setup` has been called successfully, otherwise we'll still get an error message referring to a non-existent file such as
```
# ipa-backup   --log-file /tmp2/foo
[Errno 2] No such file or directory: '/tmp2/foo'
The ipa-backup command failed. See /var/log/ipabackup.log for more information
```

Reply:
I tested that specifically and it worked fine for me but I'll take another look.

Reply:
Move the init past the __setup_logging.